spec/controllers/ems_common_controller_spec.rb

```diff
@@ -169,7 +169,24 @@
 end
 
 describe EmsContainerController do
```

Comment:
Not to be fixed in this PR, but why is `EmsContainerController` being described and tested in `ems_common_controller_spec.rb`??

cc @chrisarcand